### PR TITLE
Moved paypal_client_id to secrets manager

### PR DESCRIPTION
**Pull request by claudianguyen**
Moved paypal_client_id to secrets manager. This variable is also going to be removed in circle ci's environment variables

### CODE REVIEW

File: src/main/java/com/tucklets/app/configs/SecretsConfig.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.tucklets.app.configs;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (ShahilZ) [inline src/main/java/com/tucklets/app/configs/SecretsConfig.java:6]: `@Configuration` is better. Please update `AWSConfig` too (I forgot to change it)

File: src/main/java/com/tucklets/app/configs/SecretsConfig.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SecretsConfig {
+    private final String payPalClientId;
+
+    /* There are the variables stored in AWS Secrets Manager*/
+    public SecretsConfig(
+            @Value("${paypal_client_id}") String payPalClientId)
```
Comment (ShahilZ) [inline src/main/java/com/tucklets/app/configs/SecretsConfig.java:12]: Combine into one line?

File: src/main/java/com/tucklets/app/configs/SecretsConfig.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        this.payPalClientId = payPalClientId;
+    }
+
+    public String getPayPalClientId() { return payPalClientId; }
```
Comment (ShahilZ) [inline src/main/java/com/tucklets/app/configs/SecretsConfig.java:17]: Multi-line please

File: src/main/java/com/tucklets/app/controllers/MainPageController.java [hunk 4]
```diff
@@ -28,7 +32,7 @@
     @GetMapping("/fetchConfigs")
     public Map<String, String> fetchConfigs() {
         Map<String, String> configsResponse = new HashMap<>();
-        configsResponse.put("paypal_client_id", System.getenv("PAYPAL_CLIENT_ID"));
+        configsResponse.put("paypal_client_id", secretsConfig.getPayPalClientId());
```
Comment (ShahilZ) [inline src/main/java/com/tucklets/app/controllers/MainPageController.java:35]: Niiiiice

**Review by ShahilZ — COMMENTED** [review]
Niiiiiiice work

File: src/main/java/com/tucklets/app/configs/SecretsConfig.java (hunk 2 above)
Comment (claudianguyen) [reply src/main/java/com/tucklets/app/configs/SecretsConfig.java]: sure thing

File: src/main/java/com/tucklets/app/configs/SecretsConfig.java (hunk 1 above)
Comment (claudianguyen) [reply src/main/java/com/tucklets/app/configs/SecretsConfig.java]: sure thing

**Pull request merged**